.circleci/config.yml

```
@@ -200,6 +200,18 @@ jobs:
           registry-url: $GCR_REGISTRY_URL
           image: initcontainer-provision-relay-maintainer
           tag: latest
+  create_sortition_pool:
+    executor: docker-node
+    steps:
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: Create sortition pool
+          working_directory: ~/project/solidity
+          command: |
+            cp ./scripts/create_sortition_pool.js ./
+            node create-sortition-pool.js
```

Do we have to do `cp`?
```suggestion
          working_directory: ~/project/solidity/scripts
          command: node create-sortition-pool.js
```

---

I ran down this road, but assumed there would be a problem with sourcing  the script directly from the `/scripts` dir, since the script is currently setup in a way to lean on the installed packages in the `/solidity` dir.  

I could update the paths for each requires, or provide a separate package file for this script.  All of that seemed less straight forward than just moving the file up a level to lean on the existing packages.